content/recipes/interoperability/bioactivity-profile.md

```diff
@@ -0,0 +1,221 @@
+
+* __Ontology of units of Measure (OM)__
+ [http://www.ontology-of-units-of-measure.org/resource/om-2](http://www.ontology-of-units-of-measure.org/resource/om-2) 
+ The OM ontology provides classes, instances, and properties that represent the different concepts used for defining and using measures and units. It includes, for instance, common units such as the SI units meter and kilogram, but also units from other systems of units such as the mile or nautical mile.
+
+
+### Annotation with vocabularies
+
+With the vocabularies above, it’s possible to annotate some metadata fields automatically. For example, the script below annotates “organism: Homo sapiens” with ZOOMA.
+
+
+    ```python
+
+
+    import requests
+    zooma_root = "https://www.ebi.ac.uk/spot/zooma/v2/api/services/annotate?propertyValue="
+    query_keyword = "Homo sapiens"
+    vocab_filter = "&filter=ontology:["
+    vocab_target = "ncbitaxon"
+    query_api = zooma_root + query_keyword + vocab_filter + vocab_target + "]"
+    request_details = requests.get(query_api)
+    request_details.json()
+    ```
+
+
+### Exemplar Bioactivity datasets
+
+[SARS CoV2 phenotypic assay from Caco2 cell line](https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL4303806/)
+
+The present dataset is a subset of CARE dataset with compounds tested on the Caco2 cell line instead of on the VERO-E6 cell like in the CARE dataset. The dataset can be downloaded and beside structural information it will contain readout numbers for activity (e.g. either percentage of cellular cytopathic inhibition at one concentration or corresponding dose-response extracted IC50 (inhibition concentration at 50% inhibition)).
+
+> These recommendations are based on ChEMBL ontology requirements; similar US data bank PubChem have different ontology requirements for upload but provide a wizard-based upload process described in this blog:  [https://pubchemblog.ncbi.nlm.nih.gov/tag/pubchem-upload/](https://pubchemblog.ncbi.nlm.nih.gov/tag/pubchem-upload/)
+
+
+## Authors:
+<table>
+  <tr>
+   <td>Name
+   </td>
+   <td>Affiliation
+   </td>
+   <td>orcid
+   </td>
+   <td> CrediT role
+   </td>
+  </tr>
+  <tr>
+   <td>Andrea Zaliani
+   </td>
+   <td>Fraunhofer ITMP Hamburg
+   </td>
+   <td><a href="https://orcid.org/0000-0002-1740-8390">0000-0002-1740-8390</a>
+   </td>
+   <td>Writing - Original Draft
+   </td>
+  </tr>
+  <tr>
+   <td>Fuqi
+   </td>
+   <td>EBI
+   </td>
+   <td><a href="http://europepmc.org/search?query=AUTHORID:%220000-0002-5923-3859%22&sortby=Date">0000-0002-5923-3859</a>
+   </td>
+   <td>Writing - Original Draft
+   </td>
+  </tr>
```

Comment:
Use directive instead but I will fix it :)

Reply:
updated